CONTRIBUTING.md

```diff
@@ -96,7 +96,7 @@ We want your work to be readable by others; therefore, we encourage you to note
 
   ```bash
   python3 -m pip install ruff  # only required the first time
-  ruff .
+  ruff check .
```

Comment:
```suggestion
  ruff check
```
Looks like we don't need to specify the current directory anymore, since ruff will run in the current directory by default: https://docs.astral.sh/ruff/tutorial/#getting-started